Algorithms/Graphs/MST/prim2.cpp

```diff
@@ -0,0 +1,94 @@
+
+/*  The following is an implementation of Prim's algorithm
+    using adjacency list representation of a graph. The data structure
+    min heap is used for this. There are, again, two implementations
+    in this, the min heap code can be written differently in a class, or
+    the priority_queue present in STL can be used. Here, it's the
+    latter.
+    Now, a priority_queue is essentially a max heap, but can be used
+    as given below to form a min heap. The loop statements execute
+    v+e times, in addition to log(v) time for adding to priority_queue.
+    Overall, O(v+e)*O(log(v)) = O((e+v)*log(v)) = O(e*log(v))
+    Since for a connected graph, v = O(e).
+*/
+
+#include<bits/stdc++.h>
+#define INF INT_MAX
+
+using namespace std;
+
+int main(){
+    cout<<"Enter number of vertices of the graph"<<endl;
+    int v;
+    cin>>v;
+
+    vector<pair<int, int> > adj[v];
+    int i, j; //Iterators
+
+    cout<<"Enter number of edges of the graph"<<endl;
+    int e;
+    cin>>e;
+
+    cout<<"Enter the vertices and their edge weights in the following format :"<<endl;
+    cout<<"Vertex1 Vertex2 Weight"<<endl;
+    cout<<"PS : First vertex should be 0."<<endl;
+
+    for (i = 0; i < e; i ++){
+        int v1,v2,w;
+        cin>>v1>>v2>>w;
+        adj[v1].push_back(make_pair(w,v2));
+        adj[v2].push_back(make_pair(w,v1));
+    }
+
+    int mst[v];
+    //Array to store MST
+    int keys[v];
+    //Key values of every edge
+    bool included[v];
+    //The corresponding vertex has already been included if true
+
+    for (i = 0; i < v; i ++){
```

Comment:
Don't mix several notations in the code, in line 50 you are using `i ++` in the for loop, whereas in lines 76 and 90, you are using `++ i` in the for loop.
Please fix to one notation. This is a good practice.